### PR TITLE
Declares default font stack to prevent inheritance (specifically for …

### PR DESCRIPTION
**Pull request by mattconvente**
…icon fonts that may cascade to hint text).

### CODE REVIEW

File: src/hint-variables.scss [hunk 1]
```diff
@@ -5,6 +5,9 @@
 // Prefix for all classes. By default, BEM naming convention is used
 $prefix: 'hint--' !default;
 
+// default font family
+$fontFamily: 'Helvetica Neue', Helvetica, Arial, sans-serif;
```
Comment (chinchang) [inline src/hint-variables.scss]: Missing `!default` after variable value.

File: src/hint-core.scss [hunk 2]
```diff
@@ -13,6 +13,7 @@
 .hint, [data-hint] {
 	position: relative;
 	display: inline-block;
+    font-family: $fontFamily;
```
Comment (chinchang) [inline src/hint-core.scss]: Indentation wrong.

File: hint.base.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/*! Hint.css (base version) - v1.3.5 - 2015-06-16
```
Comment (chinchang) [inline hint.base.css]: @mattconvente Please remove this any built CSS file for the PR. I'll build this when doing final merge.

**Comment by chinchang** [comment]
@mattconvente Thanks for this PR...this is really helpful. I have mentioned few comments. Please address and let me know so we can merge this.

**Comment by mattconvente** [comment]
@chinchang Changes updated, please see latest commits. Had some trouble with spaces —> tabs so just duplicated the previous line and changed its contents.

File: src/hint-core.scss [hunk 4]
```diff
@@ -13,7 +13,7 @@
 .hint, [data-hint] {
 	position: relative;
 	display: inline-block;
-
+	font-family: $fontFamily;
```
Comment (chinchang) [inline src/hint-core.scss]: `[data-hint]` doesn't need to have font family set. It needs to be on the `:after` element inside it.

**Comment by joaocunha** [comment]
@mattconvente do you want me to take this over?

**Comment by chinchang** [comment]
closing due to inactivity. Will take this up myself.

**Pull request closed without merging**